README.md

```diff
@@ -21,8 +21,7 @@ https://github.com/scality/Guidelines/blob/master/CONTRIBUTING.md).
 This repository will include installation resources to deploy the full Zenko
 stack over different orchestration systems.
 
-We started with [Docker Swarm](https://docs.docker.com/engine/swarm/) but others
-like [Kubernetes](https://kubernetes.io/) will follow.
+We started with [Docker Swarm](https://docs.docker.com/engine/swarm/) and [Helm charts](https://github.com/scality/Zenko/blob/master/charts/README.md) for [Kubernetes](https://kubernetes.io/). The following information is about the Docker Swarm setup.
```

Comment:
I would rather this points to metal-k8s, our custom setup process for kubespray, to support long-term service deployments on bare metal, since that's what we recommend using :) https://github.com/scality/metal-k8s